app/ui/src/app/app.component.html

```diff
@@ -10,7 +10,6 @@
         alt="" />
       <img *ngIf="logoDarkBg" [class]="productBuild ? 'navbar-brand-name product' : 'navbar-brand-name'" [src]="loggedIn ? logoDarkBg : logoWhiteBg"
         alt="{{ appName }}" />
```

Comment:
Should this `alt` also be changed to something else?

Reply:
no that should be fine